core/domain/activity_domain.py

```diff
@@ -28,17 +28,30 @@ class ActivityReference(object):
     """
```

Comment:
@seanlip If mentioning the attributes in the doc string for a class, is it necessary to repeat the same inside the doc strings of a module in the same class ?

```python
class ActivityReference(object):
    """Domain object for an activity reference.
    An activity is a piece of learning material that can be created in Oppia.
    Currently, the only available types of activities are explorations and
    collections.
    Attributes:
        activity_type: A string indicating the activity type.
        activity_id: A string indicating the activity id.
    """

    def __init__(self, activity_type, activity_id):
        """Constructs an ActivityReference domain object.
        Args:
            activity_type: str. The activity type.
            activity_id: str. The activity id.
        """
        self.type = activity_type
        self.id = activity_id
```
Here activity_type and activity_id are having duplicate mentions in class as well as inside method.